Storage/index.js

```diff
@@ -208,32 +299,38 @@ async function getModels() {
         contributions: DataTypes.DECIMAL(10,2),
         expenditures: DataTypes.DECIMAL(10,2)
     });
+    Candidacy.hasOne(CampaignFinance, {
+        foreignKey: 'candidacy_id'
+    });
+    CampaignFinance.belongsTo(Candidacy, {
+        foreignKey: 'candidacy_id'
+    });
 
-    const CandidateSearch = sequelize.define('CandidateSearch', {
```

Comment:
Can we put candidate search back for now? The analytics tool I built is based on its existence. Is there any harm in having this model?